src/components/package-search-list.js

```diff
@@ -21,7 +21,7 @@ function PackageSearchList({
       fetch={fetch}
       collection={collection}
       onUpdateOffset={onUpdateOffset}
-      itemHeight={80}
+      itemHeight={70}
```

Comment:
Can we move the hardcoded heights to constants?

Reply:
Sure, commit added.